<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSettingsWidget</class>
 <widget class="QWidget" name="ChannelSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="1" column="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="channelSel">
     <property name="title">
      <string>Channel Selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="channel">
        <item>
         <property name="text">
          <string>1 - centre freq. 3494.4 GHz (500 MHz BW)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 - centre freq. 3993.6 GHz (500 MHz BW)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 - centre freq. 4492.8 GHz (500 MHz BW)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 - centre freq. 3993.6 GHz (900 MHz BW)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 - centre freq. 6489.6 GHz (500 MHz BW)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7 - centre freq. 6489.6 GHz (900 MHz BW)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>SFD Type</string>
     </property>
     <widget class="QCheckBox" name="nsSFD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>nSFD</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="preambleLength">
     <property name="title">
      <string>Preamble Length (symbol repetitions)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="l64">
        <property name="text">
         <string>64</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="l128">
        <property name="text">
         <string>128</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="l256">
        <property name="text">
         <string>256</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="l512">
        <property name="text">
         <string>512</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="l1024">
        <property name="text">
         <string>1024</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="l1536">
        <property name="text">
         <string>1536</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="l2048">
        <property name="text">
         <string>2048</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="l4096">
        <property name="text">
         <string>4096</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="PRF">
     <property name="title">
      <string>PRF (Pulse Repetition Freq.)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QRadioButton" name="prf64">
        <property name="text">
         <string>64 MHz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="prf16">
        <property name="text">
         <string>16 MHz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QGroupBox" name="pCodeSel">
     <property name="title">
      <string>Preamble Code</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="pCode"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="ok">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QGroupBox" name="useCases">
     <property name="title">
      <string>Use Cases</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="usecase">
        <item>
         <property name="text">
          <string>1 - User Defined</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 - Long Range</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 - Max Density</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 - Defaults</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QGroupBox" name="dataRate">
     <property name="title">
      <string>Data Rate (bits/s)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QRadioButton" name="dr850">
        <property name="text">
         <string>850k</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="dr6M8">
        <property name="text">
         <string>6.81M</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="dr110">
        <property name="text">
         <string>110k</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
